interface-definitions/service_ssh.xml.in

```diff
@@ -275,6 +275,24 @@
               </constraint>
             </properties>
           </leafNode>
+          <leafNode name="trusted-user-ca-key">
+            <properties>
+              <help>Trusted user CA key</help>
+              <valueHelp>
+                <format>filename</format>
```

Comment:
Instead of using a file path, this should be using the builtin PKI config for key storage.

Example how to use here: https://github.com/vyos/vyos-1x/blob/current/python/vyos/ifconfig/interface.py#L1563-L1577

Reply:
Hi sarthurdev-san,  
Thank you for your review!

Oh! It seems that using the PKI commands would be the better approach! Thank you for pointing that out.
https://docs.vyos.io/en/latest/configuration/pki/index.html  

Based on your feedback, it seems better to implement the feature for retrieving and installing CA certificates from remote locations as a separate PKI subcommand. I’ll address this in a future PR rather than including it in this one. I’m thinking of a format like
`import pki ca newca remote http://path/ca.pem`

For this PR, I will proceed with using the PKI management mechanism you suggested. Specifically, I plan to leverage certificates added via the command:  
`import pki ca newca file /path/ca.pem`

Reply:
Hi. @sarthurdev san
Would it be possible for you to continue reviewing?
I would greatly appreciate it!